draft-looker-oauth-jwt-cwt-status-list.md

```diff
@@ -39,20 +42,28 @@ The status list data structures themselves are also represented as JWTs or CWTs.
 
-This document defines Status List representations in JWT and CWT formats that describe the individual statuses of multiple Referenced Tokens, which themselves are also JWTs or CWTs. The statuses of all Referenced Tokens are conveyed via a bit array in the Status List. Each Referenced Token is allocated an index during issuance that represents its position within this bit array. The value of the bit(s) at this position correspond to the Referenced Token's status. The document also defines how an issuer of a Referenced Token references a Status List Token. Status Lists may be composed for expressing a range of Status Types. This document defines basic Status Types for the most common use cases as well as an extensibility mechanism for custom Status Types. The Status List Token may be used by an issuer in the Issuer-Holder-Verifier model to express the status of verifiable credentials (Referenced Tokens) issued by an issuer.
+This document defines Status List representations in JSON or CBOR formats and a Status List Token representation in JWT or CWT formats that describe the individual statuses of multiple Referenced Tokens, which themselves are also JWTs or CWTs. The statuses of all Referenced Tokens are conveyed via a bit array in the Status List. Each Referenced Token is allocated an index during issuance that represents its position within this bit array. The value of the bit(s) at this position correspond to the Referenced Token's status. The document also defines how an issuer of a Referenced Token references a Status List (Token). Status Lists may be composed for expressing a range of Status Types. This document defines basic Status Types for the most common use cases as well as an extensibility mechanism for custom Status Types. The Status List Token may be used by an issuer in the Issuer-Holder-Verifier model to express the status of verifiable credentials (Referenced Tokens) issued by an issuer.
 
 The following diagram depicts the basic conceptual relationship.
 
 ~~~ ascii-art
 
-+------------------+                    +-------------------+
-|                  |     References     |                   |
-|                  |------------------->|                   |
-| Referenced Token |                    | Status List Token |
-|   (JWT or CWT)   |                    |    (JWT or CWT)   |
-|                  |  Describes Status  |                   |
-|                  |<-------------------|                   |
-+------------------+                    +-------------------+
++-------------------+                  +------------------------+
+|                   | describes status |                        |
+|    Status List    +------------------> Status Reference Token |
```

Comment:
```suggestion
|    Status List    +------------------>      Referenced Token     |
```
                                   
"Status Reference Token" is really confusing. While "Referenced Token" is used elsewhere in the draft.